.asf.yaml

```diff
@@ -28,7 +28,7 @@ github:
   del_branch_on_merge: true
   #labels:
   enabled_merge_buttons:
-    merge:  false
+    merge:  true
```

Comment:
Prefer to disable merge button.
In order to keep a clear history, we do not expect to introduce merge commits.

Reply:
Temporarily enable for migrating commons module, disable it after merge #2628... cc @imbajin

Reply:
> Temporarily enable for migrating commons module, disable it after merge #2628... cc @imbajin

Yes, just temporarily open, then close again

Reply:
get it

Reply:
> get it

@javeme need your approve to merge this PR 🔢